src/log/Log.cc

```diff
@@ -327,7 +332,7 @@ void Log::_flush(EntryQueue *t, EntryQueue *requeue, bool crash)
       }
 
       if (do_stderr) {
-        cerr << buf << std::endl;
+        cerr << m_log_stderr_prefix << buf << std::endl;
```

Comment:
missing space between prefix and buf?

Reply:
but we can always have a trailing space (or a colon =) in the configured prefix. i think the configurable prefix offers us more flexibility.

Reply:
True, we can. But I very much doubt users will ever do that on the first pass. Until they realize that they require to have a space in the prefix. I'd say that's not very intuitive, even if properly documented.

Reply:
hmm, makes sense from the usability perspective. but i think the target user of this feature will be a containerized Ceph. and i assume that the number of developers who write the ansible playbook (or whatever script to deploy Ceph in the container) is much less than that of our typical user who deploys Ceph use ceph-deploy. so it's acceptable to do so, i guess.

anyway, i don't feel strong either way.

Reply:
There's no space because by default the prefix is an empty string.